tests/golden/check-symlinks/expected1.gold

```diff
@@ -10,10 +10,15 @@
             - text: "Symlink ref"
             - link: ../s.md
             - anchor: -
-        - reference (relative) at src:11:1-36:
-            - text: "Symlink ref with anchor"
-            - link: ../s.md
-            - anchor: a
+    - anchors:
+        none
+
+  s.md:
+    - references:
+        - reference (relative) :
+            - text: ""
+            - link: dir/b.md
+            - anchor: -
```

Comment:
This is how a symlink reference looks like by using the plain system that we already have. It can be improved to show in some way that `s.md` is a symlink that points to `dir/b.md`.